server/services/container.ts

```diff
@@ -0,0 +1,40 @@
+type FilterParams<T> = (value: T, index: number, array: T[]) => unknown
+type MapParams<T, R> = (t: T, index: number, array: T[]) => R
+type ExcludedItem = [item: unknown, reason: string]
+
+export default class Container<T> {
```

Comment:
Didn't really know what to call this.. `Caseload` was tempting but it can contain anything really.
It's used quite a few places so wanted to call it something short... rather than `ExclusionRecordingList` or something